### PR TITLE
added possibility to set timeout from grafana webhook

### PR DESCRIPTION
**Pull request by kjetilmjos**
Is there any way of using the default alert timeout setting from the config file instead of hardcoding the fallback value to 86400?

### CODE REVIEW

File: alerta/webhooks/grafana.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def parse_grafana(alert, match, args):
     group = args.get('group', 'Performance')
     origin = args.get('origin', 'Grafana')
     service = args.get('service', 'Grafana')
+    timeout = args.get('timeout', 86400)
```
Comment (satterly) [inline alerta/webhooks/grafana.py]: This should work, but if you could test it that would be good...
```
timeout = args.get('timeout', current_app.config('ALERT_TIMEOUT'))
```

Comment (kjetilmjos) [reply alerta/webhooks/grafana.py]: Tested it now. Got and error saying:

2018-08-27 17:38:25,022 - flask.app[93]: ERROR - 'Config' object is not callable [in /venv/lib/python3.6/site-packages/alerta/exceptions.py:95]
Traceback (most recent call last):
  File "/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/venv/lib/python3.6/site-packages/flask_cors/decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/venv/lib/python3.6/site-packages/alerta/auth/decorators.py", line 37, in wrapped
    return f(*args, **kwargs)
  File "/venv/lib/python3.6/site-packages/alerta/webhooks/grafana.py", line 73, in grafana
    incomingAlert = parse_grafana(data, match, request.args)
  File "/venv/lib/python3.6/site-packages/alerta/webhooks/grafana.py", line 33, in parse_grafana
    timeout = args.get('timeout', current_app.config('ALERT_TIMEOUT'))
TypeError: 'Config' object is not callable

**Comment by satterly** [comment]
Sorry, that should have been square brackets to index into a dictionary...
```
timeout = args.get('timeout', current_app.config['ALERT_TIMEOUT'])
```

**Comment by kjetilmjos** [comment]
Changed the file to brackets. Tested and verified the new timeout setting with Grafana

**Comment by satterly** [comment]
Good stuff. Thanks a lot. 👍

**Pull request merged**